app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::V1::CommentsController < ApplicationController
+end
+
+class Api::V1::CommentsController < ApplicationController
+  load_and_authorize_resource
+  rescue_from CanCan::AccessDenied do |_exception|
+    redirect_to root_path, notice: 'Access denied'
+  end
+
+  def create
+    @new_comment = Comment.new(comment_params)
+    @new_comment.post = Post.find(params[:id])
+    @new_comment.author = current_user
+
+    if @new_comment.save
+      render json: @new_comment, status: :created, location: @new_comment
+    else
+      render json: @new_comment.errors, status: :unprocessable_entity
+    end
+  end
+
```

Comment:
- [ ] According to the project requirements, you should add a method here that gets all comments for a post. On this controller, I only see a method that creates comments on a post.